app/views/admin/events/_form.html.haml

```diff
@@ -0,0 +1,77 @@
+.row
+  .col-md-12
+    .page-header
+      %h1
+        New Event
+.row
+  .col-md-12
+    = semantic_form_for(@event, url: @url) do |f|
+      = f.inputs name: 'Event Information' do
```

Comment:
Couldn't we use the existing proposals/proposal_form with minor adjustments and add speaker and submitter at the bottom in the admin/events/_form ?